packages/modules/devices/huawei_smartlogger/config.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+from helpermodules.auto_str import auto_str
+from modules.common.component_setup import ComponentSetup
+
+
+@auto_str
+class Huawei_SmartloggerConfiguration:
+    def __init__(self, ip_address: Optional[str] = None):
+        self.ip_address = ip_address
+
+
+@auto_str
+class Huawei_Smartlogger:
+    def __init__(self,
+                 name: str = "Huawei_Smartlogger",
+                 type: str = "huawei_smartlogger",
```

Comment:
```suggestion
class HuaweiSmartlogger:
    def __init__(self,
                 name: str = "Huawei Smartlogger",
                 type: str = "huawei_smartlogger",
```
Naming bitte anhand der Python-Konvention: Klassen mit CamelCase (ohne Unterstrich)